src/rvya25-profile.adoc

```diff
@@ -0,0 +1,466 @@
+exceptions on attempts to execute unimplemented opcodes.
+
+=== RVYA25S64 Profile
+
+The RVYA25S64 profile specifies the ISA features available to a
+supervisor-mode execution environment in 64-bit applications
+processors.  RVYA25S64 is based on privileged architecture version
+1.13.
+
+==== RVYA25S64 Mandatory Base
+
+RVY64I is the mandatory base ISA for RVYA25S64 and is little-endian.
+The `ECALL` instruction operates as per the unprivileged architecture
+specification.  An `ECALL` in user mode causes a contained trap to
+supervisor mode.  An `ECALL` in supervisor mode causes a requested
+trap to the execution environment.
+
+==== RVYA25S64 Mandatory Extensions
+
+The following unprivileged extensions are mandatory:
+
+- The RVYA25S64 mandatory unprivileged extensions include all the
+mandatory unprivileged extensions in RVYA25U64.
+
+- *Zifencei*  Instruction-Fetch Fence.
+
+NOTE: Zifencei is mandated as it is the only standard way to support
+instruction-cache coherence in RVYA25 application processors.  A new
+instruction-cache coherence mechanism is under development
+(tentatively named Zjid) which might be added as an option in the
+future.
+
+The following privileged extensions are mandatory:
+
+- *Ss1p13*  Supervisor architecture version 1.13.
+
+- *Svbare* The `satp` mode Bare must be supported.
+
+- *Sv39* Page-based 39-bit virtual-Memory system.
+
+- *Svade* Page-fault exceptions are raised when a page is accessed
+   when A bit is clear, or written when D bit is clear.
+
+- *Ssccptr* Main memory regions with both the cacheability and
+   coherence PMAs must support hardware page-table reads.
+
+- *Sstvecd* `stvec.MODE` must be capable of holding the value 0
+  (Direct).  When `stvec.MODE=Direct`, `stvec.BASE` must be capable of
+  holding any valid four-byte-aligned address.
+
+- *Sstvala* `stval` must be written with the faulting virtual address
+  for load, store, and instruction page-fault, access-fault, and
+  misaligned exceptions, and for breakpoint exceptions other than
+  those caused by execution of the `EBREAK` or `C.EBREAK` instructions.
+  For virtual-instruction and illegal-instruction exceptions, `stval` must be written with the
+  faulting instruction.
+
+- *Sscounterenw* For any `hpmcounter` that is not read-only zero, the
+  corresponding bit in `scounteren` must be writable.
+
+- *Svpbmt* Page-based memory types
+
+- *Svinval* Fine-grained address-translation cache invalidation.
+
+- *Svnapot* NAPOT translation contiguity.
+
+- *Sstc* supervisor-mode timer interrupts.
+
+- *Sscofpmf* count overflow and mode-based filtering.
+
+- *Ssu64xl* `sstatus.UXL` must be capable of holding the value 2
+(i.e., UXLEN=64 must be supported).
+
+- *Sha* The augmented hypervisor extension.
+
+NOTE: Sha is defined in the RVA23 profile document.
+
+The following mandatory extension is RVY25S64 specific:
+
+- *Svy* Read/write access control for capability tags in the page table entries.
```

Comment:
I thought we weren't inventing new names for things that are part of the base ISA when S-mode is present?

Reply:
I believe we've removed all the other ones, but this one I am not sure if ARC wants an explicit name since it does not apply to all translation schemes (Sv32). I will explicitly ask about this one when I send the review email.

Reply:
I don't think it's helpful to frame something as a separate extension when it's not separable from the thing it's extending